index.html

```diff
@@ -120,5 +120,40 @@ <h5>Next level iceland blue bottle fingerstache:</h5>
         </ul>
       </section>
     </section>
+    <section class="contact">
+      <h2>contact</h2>
+      <form>
+        <label for="name">name</label>
+        <input type="text" id="name">
+        <label for="email">e-mail</label>
+        <input type="email" id="email">
+        <label for="topic">topic</label>
+        <input type="text" id="topic">
+      </form>
+      <img src="assets/office.jpg" alt="office building image">
+      <p>
+        Our office is located<br/>
+        in the heart of Warsaw
+      </p>
+      <p>Feel free to say hello!</p>
+      <p>
+        Warsaw St. 23/21<br/>
+        03-323 Warsaw,<br/>
+        POLAND
+      </p>
+    </section>
+    <footer>
+      <!-- assets/socialmedia.svg as a background image for footer -->
```

Comment:
Background image or just a text in the background?

Reply:
Hmm, I found the svg for that so I thought I should use it like that. i also thought about a span with the text. Would you opt for text?

Reply:
If it is the font-family that we are already using then yes. If it is only a visual thing it can be always using `aria-hidden`

Reply:
Should I be able to find this information about font-family in the svg file? I can't find it 😭

Reply:
But at a guess, visually, it looks the same font-family as headings.

Reply:
The svg file has probably outlines, not text nodes (each letter transformed to use paths). It looks like the same font as the rest of the page.